manifests/m/Microsoft/PowerShell/7.4.5.0/Microsoft.PowerShell.installer.yaml

```diff
@@ -4,7 +4,6 @@
 PackageIdentifier: Microsoft.PowerShell
 PackageVersion: 7.4.5.0
 MinimumOSVersion: 10.0.17763.0
-InstallerType: wix
 Scope: machine
```

Comment:
I believe this should be moved down for Wix installers only, since I believe MSIX is user-scoped only

Reply:
@mdanish-kh oh I didn't know that, thanks! Does that placement look correct now? Does it matter that our previous manifests didn't have the scope corrected like that for when users upgrade?

Reply:
I'll resolve this conversation to expedite the `arm` availability. If there's any issue, please let me know.

Reply:
@stephengillie yep that was the right call, thanks!